### ISSUE TITLE
How to fix? :(

### ISSUE THREAD

**Issue by chadfurman** [mariusGundersen/chrome-flexbox-bug#1]
Yes, I have this bug locally... how to fix it though?  Haha.  You only made me cry harder.

**Comment by chadfurman** [comment]
lol uh, setting min-height:0; on .text-container works....

**Comment by mariusGundersen** [comment]
Yes, throwing min-height and min-width at pretty much every element solves this. Sorry, this repo wasn't for finding the solution, it's for pointing out the problem.

**Comment by chadfurman** [comment]
seems you only have to throw it on the container element of the thing that's overflowing.
